service.go

```diff
@@ -7,7 +7,8 @@ import (
 )
 
 type ServiceId struct {
-	Id graphql.ID `json:"id"`
+	Id      graphql.ID `json:"id"`
+	Aliases []string   `json:"aliases,omitempty"`
 }
 
 type Service struct {
```

Comment:
We need to remove this from the `Service` struct since it inherits from `ServiceId`

Reply:
Great catch, I didn't see it was already using ServiceId.  I'll update it now.